user-guide/modules/ROOT/pages/boost-history.adoc

```diff
@@ -48,9 +48,7 @@ These libraries, among others, showcase the value and versatility of Boost in pr
 
 == Current Status
 
-The Boost library is managed by the https://cppalliance.org/[pass:[C++] Alliance], with a mission "to make the C++ programming language accessible and useful to anyone who wishes to learn and apply the language."
-
-The Boost pass:[C++] Library has come a long way since its inception in the late 1990s, to a widely used and influential https://www.boost.org/doc/libs/[collection of over 180 libraries]. Boost has played a crucial role in shaping the pass:[C++] language and its ecosystem. As the pass:[C++] language continues to evolve, the Boost pass:[C++] Library will undoubtedly remain an essential and innovative part of its story.
+Boost pass:[C++] has come a long way since its inception in the late 1990s, to a widely used and influential https://www.boost.org/doc/libs/[collection of over 180 libraries]. Boost has played a crucial role in shaping the pass:[C++] language and its ecosystem. As the pass:[C++] language continues to evolve, the Boost pass:[C++] Library will undoubtedly remain an essential and innovative part of its story.
```

Comment:
> Boost pass:[C++]

I never heard the term "Boost C++". It's not what it's called.

It's either "The Boost Libraries" or just "Boost".